### PR TITLE
Saffet responsive grid layout

### PR DESCRIPTION
**Pull request by Saffet01**

### CODE REVIEW

File: responsive-grid-layout/saffet/global.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 * {
   box-sizing: border-box;
   font-family: "Segoe UI Adjusted", "Segoe UI", "Liberation Sans", sans-serif;
+  overflow: hidden;
```
Comment (annasvst) [inline responsive-grid-layout/saffet/global.css:4]: Why do you feel a need to specify overflow property in this case?

**Review by annasvst — APPROVED** [review]
Looks good Saffet, approved!

File: responsive-grid-layout/saffet/global.css (hunk 1 above)
Comment (Saffet01) [reply responsive-grid-layout/saffet/global.css:4]: Because there was a scroll bar and I didn't want to see any scroll bar on right side

Comment (annasvst) [reply responsive-grid-layout/saffet/global.css:4]: Vertical scroll bar is expected in a layout like this as the page should be higher than the height of the screen so in this case `overflow: hidden;` is not relevant in this case.

**Pull request merged**